### PR TITLE
Remove MultiJSON dependency

### LINKED ISSUE #4: Remove MultiJSON dependency as it's deprecated
See: https://github.com/rails/rails/pull/10576

Probably want to replace it with stdlib's JSON.

May want to check what Faraday is using nowadays.

### PR DESCRIPTION
**Pull request by turino**
Everything seems to work when I remove the dependency. Not sure whether it's important to have `require "json"` stated explicitly somewhere.

closes #4

### CODE REVIEW

File: lib/fmrest/spyke/json_parser.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "json"
```
Comment (turino) [inline lib/fmrest/spyke/json_parser.rb]: I have no idea if this is needed, or if this is the place to put it. Everything works if we leave this line out. Presumably Faraday is loading the JSON library for us (and other gems as well?), but maybe it's not good to depend on other gems to load required libraries?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 require "bundler/setup"
 
 require "spyke"
-require "multi_json"
```
Comment (turino) [inline spec/spec_helper.rb]: again, not sure whether I should explicitly put `require "json"` here. Presumably if we're requiring it explicitly above, it's not needed here, right?

File: lib/fmrest/spyke/json_parser.rb [hunk 3]
```diff
@@ -191,11 +193,7 @@ def delete_request?(env)
       # @param source [String] a JSON string
       # @return [Hash] the parsed JSON
       def parse_json(source)
-        if defined?(::MultiJson)
-          ::MultiJson.load(source, symbolize_keys: true)
-        else
-          ::JSON.parse(source, symbolize_names: true)
-        end
+        JSON.parse(source, symbolize_names: true)
```
Comment (turino) [inline lib/fmrest/spyke/json_parser.rb]: there's no naming conflict for `JSON`, right? So do we need `::JSON` here? I'm fine to leave it in, if that's standard convention.

Comment (pilaf) [reply lib/fmrest/spyke/json_parser.rb]: The `::` is there just to ease my paranoia of "what if we later on create a module named `JSON` inside the `FmRest` namespace?", which of course wouldn't really happen in practice.

File: lib/fmrest/spyke/json_parser.rb (hunk 1 above)
Comment (pilaf) [reply lib/fmrest/spyke/json_parser.rb]: Let's leave it in.

**Pull request merged**